integrations/tailwindcss-cli/tests/cli.test.js

```diff
@@ -291,6 +291,7 @@ describe('Build command', () => {
            -i, --input              Input file
            -o, --output             Output file
            -w, --watch              Watch for changes and rebuild as needed
+           -p, --poll               Poll for changes every so often and rebuild
```

Comment:
Do you think something like "Use polling instead of filesystem events when watching" would be a better description here?

Reply:
Yes, yes I do.